.github/ISSUE_TEMPLATE/lesson_contribution.md

```diff
@@ -0,0 +1,14 @@
+---
+name: New lesson material
+about: Develop lessons
+title: ''
+labels: enhancement
```

Comment:
Shouldn't this label be different from the other one ('suggestion')?

Reply:
Thanks, I separated them by adding more labels. Just like the type of issues, the list of the labels can be improved as well.